tests/unit/services/UserService.test.js

```diff
@@ -36,4 +36,19 @@ describe('Test: UserService', () => {
       expect(user.firstname).toEqual('Pradeep');
     });
   });
+
+  describe('Test function: find', () => {
+    it('should return all user information', async () => {
+      const users = [{
```

Comment:
I'd say `should return all userS information` since below you defined `const users`

Reply:
done